From 6d964e3c6d4dc07c5241603a44a79d60c83e2b14 Mon Sep 17 00:00:00 2001
From: Yan Markman <ymarkman@marvell.com>
Date: Sun, 18 Mar 2018 15:41:32 +0200
Subject: [PATCH 1321/1345] fix: net: mvpp2x: increase port statistic polling
 time

commit  dd921bc96d3b2dc6c0f3c0f3321d63e33b1eeb95 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

Statistic access is under spinlock.
Increase statistic-task polling time from 250ms to 1 sec.
This time is big enough to avoid traffic-impact but
small enough before potential Byte-Counter wrapping.

Change-Id: Id364e6db86d758b05ed8e8fb94955e742507b6d6
Signed-off-by: Yan Markman <ymarkman@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/52210
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/net/ethernet/marvell/mvpp2x/mv_pp2x.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x.h b/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x.h
index 82eee59..91bb30b 100644
--- a/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x.h
+++ b/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x.h
@@ -57,7 +57,7 @@
 
 #define IRQ_NAME_SIZE (36)
 
-#define STATS_DELAY	250
+#define STATS_DELAY	1000 /*mSec*/
 
 #define TSO_TXQ_LIMIT 100
 #define TXQ_LIMIT (MAX_SKB_FRAGS + 2)
-- 
1.7.9.5

